tests/test_const_table.py

```diff
@@ -126,6 +126,22 @@ def test_table_display_repr_empty_result(db: gp.Database):
     assert (t[lambda t: t["id"] == 0]._repr_html_()) == ""
 
 
+def test_table_display_result_null(db: gp.Database):
+    # fmt: off
+    rows = [([1,], None,), ([2,], "Tiger",), ([3,], None,), ([None,], "Fox")]
+    # fmt: on
+    t = gp.to_table(rows, db=db, column_names=["id", "animal"])
+    expected = (
+        "| id     || animal |\n"
+        "====================\n"
+        "| [1]    || None   |\n"
```

Comment:
It was my initial plan. But since the default result of `None` in `_repr_html_` is to display `"None"`, I have changed it to the current behavior for `__repr__`.

Reply:
Let me check